app/src/main/java/org/luxoft/sdl_core/BluetoothLongReader.java

```diff
@@ -27,6 +27,14 @@ public void setCallback(LongReaderCallback callback) {
         mCallback = callback;
     }
 
+    public void resetBuffer() {
```

Comment:
@AKalinich-Luxoft 
seems we could use this method in the end of method
public void processReadOperation(byte[] value)
line 68
of this class

Reply:
@VadymLuchko fixed in https://github.com/CustomSDL/sdl_android_launcher/pull/8/commits/652e482814a3d6c21375d640b0602ebf3fd222eb